modules/ROOT/pages/quick-start.adoc

```diff
@@ -8,6 +8,17 @@ Anypoint Monitoring provides data on Mule apps that are running on either CloudH
 [NOTE]
 Functional Monitoring is covered in xref:api-functional-monitoring::index.adoc[API Functional Monitoring].
 
+== Runtime Version Requirements
+
+For complete functionality, you can use the following runtime versions with Anypoint Monitoring:
+
+* Mule 3: versions 3.8.7 or newer
+* Mule 4: versions 4.1.6 or newer
+
+If you are using a Mule 4 version that is older than version 4.1.5, you cannot use built-in API metrics.
+
+Versions 3.8.6 or older are not supported. Additionally, all -AM runtime versions are deprecated.
```

Comment:
Regarding -AM runtime, I suggest saying something like "Additionally, any runtime version with a '-AM' postfix is deprecated"

Reply:
Note that I saw in the "Built-in API Analytics Dashboard" page, there's verbiage there that also lists out versions. We should scan the rest of the Monitoring docs and remove other places that lists out versions.

Reply:
```suggestion
Versions 3.8.6 or older are not supported. Additionally, all runtime versions that have the `-AM` suffix are deprecated.
```

Reply:
Yes - I believe I have a ticket for that.